tests/unit/tasks/test_tool_task.py

```diff
@@ -30,6 +30,7 @@ def test_run_with_memory(self, agent):
 
         agent.add_task(task)
 
+        agent.meta_memory = None
```

Comment:
Is this still necessary with the revert to optional thought?

Reply:
Don't think is needed anymore, deleted